features/dd-sdk-android-trace/build.gradle.kts

```diff
@@ -60,7 +52,6 @@ android {
 }
 
 dependencies {
-    coreLibraryDesugaring(libs.desugarJdk)
     api(project(":dd-sdk-android-core"))
     api(libs.openTelemetryApi)
```

Comment:
if we are extracting `OpenTelemetry` into a dedicated module, why do we keep this one here?

Reply:
just a bad copy / paste sorry